sdcm/sct_config.py

```diff
@@ -1761,6 +1761,10 @@ class SCTConfiguration(dict):
              help="Workload name, can be: write|read|mixed|unset."
                   "Used for e.g. latency_calculator_decorator (use with 'use_hdrhistogram' set to true)."
                   "If unset, workload is taken from test name."),
+
+        dict(name="alternator_running_time_multiplier", env="ALTERNATOR_RUNNING_TIME_MULTIPLIER", type=float,
```

Comment:
SCT environment variables, should start with `SCT_`

Also I don't understand what this option is exactly for, it should be documented a bit clearer.
how running time is related to load here, again, really not clear.

Reply:
it's sounds like it might be similar to `stress_duration`, but expressed in multiplier

`stress_duration` has the extra feature of setting the pipeline timeout correctly to accumulate the change in test duration, which your new flag probably lacks (hence doubling my get you to a timeout of the pipeline)